Lib/fontTools/voltLib/parser.py

```diff
@@ -0,0 +1,633 @@
+from __future__ import (
+    print_function, division, absolute_import, unicode_literals)
+from collections import OrderedDict
+import fontTools.voltLib.ast as ast
+import fontTools.feaLib.parser as parser
```

Comment:
Why does voltLib use the feaLib parser? The dependency on feaLib.parser.SymbolTable seems strange — doesn’t voltLib needs its own SymbolTable?